ci/checks/validate_docs_snippets.py

```diff
@@ -49,7 +49,7 @@ def run_grep(target_dir: str) -> List[str]:
     return res.stdout.splitlines()
 
 
-def parse_grep(lines: list[str]) -> list[str]:
+def parse_grep(lines: List[str]) -> List[str]:
```

Comment:
This was giving a type error

Reply:
If you `from __future__ import annotations` using `list` should work.